apps/backend/src/config.ts

```diff
@@ -65,6 +65,7 @@ export const config = {
     contractAddress: getAddress(env('EVM_CHAIN_CONTRACT_ADDRESS')),
     confirmations: Number(env('EVM_CHAIN_CONFIRMATIONS', '12')),
     checkInterval: Number(env('EVM_CHAIN_CHECK_INTERVAL', '30000')),
+    premiumPct: Number(env('EVM_CHAIN_PREMIUM_PCT', '4.00')), // 400% premium
```

Comment:
The naming here is confusing. If a variable has Pct or Percent in its name, I usually expect it to be divided by 100 before use (so 4.00 → 0.04 = 4%). Since the code is using it directly as a factor, maybe `priceMultiplier` would be clearer. In that case 4.0 would mean a 300% premium (final price = 4× base), which avoids the “is this 4% or 400%?” ambiguity.

Reply:
Updated to `priceMultiplier` in name and calculation

Reply:
Did you push? I don't see the update.